tests/app/main/views/test_templates.py

```diff
@@ -31,19 +31,23 @@
 
 @pytest.mark.parametrize('permissions, expected_message', (
     (['email'], (
+        'Every message starts with a template. You can change it later. '
```

Comment:
Hmm, this doesn't look right to me. Did these messages somehow get the old and new versions after my PR was merged into this?

Reply:
Yeah, looks like an artifact from the most recent merge. Annoyingly, the tests still pass.